docs/CodeDevelopment.rst

```diff
@@ -14,7 +14,7 @@ Enter the root of your source repository and execute:
 
 .. code-block:: console
 
-    $ spack dev-build  <package> @develop <variant>
+    $ spack dev-build --until build <package> @<version>
 
 This will install the package as is. The downside of this approach is that
```

Comment:
`--until build` will **_build_** the package, not **_install_** it.

Reply:
Fixed in https://github.com/C2SM/spack-c2sm/pull/943/commits/bac30be3f1bfa9040810ca81a5c1afbb10bc0d61

(I actually did that already, but forgot to save/push 🙄)